app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
+  before_action :authenticate_user!
+
+  def index
+    @rents = Rent.all
```

Comment:
Are you sure you want to pass all the rents to the user? For example, user A wants to list his/her rents, are you going to show ALL the rents in the system? Remember that, according to the apiary, the request URL has the `user_id`, maybe you can filter the methods...